api/internal/target/kusttarget_configplugin.go

```diff
@@ -281,7 +289,7 @@ var transformerConfigurators = map[builtinhelpers.BuiltinPluginType]func(
 		p := f()
 		err = kt.configureBuiltinPlugin(p, c, bpt)
 		if err != nil {
-			return nil, err
+			return nil, errors.Wrap(err, "failed to configure transformer")
```

Comment:
You can add `// nolint:wrapcheck` to the end of this line to bypass the lint error. I'm not sure why it's still failing.